index.js

```diff
@@ -118,7 +193,8 @@ function myMap(array, callback) {
 // // => ["Hi, Raffy!", "Hi, Chase!"]
 // console.log("%c----------", "color: red") 
 
-
+function greet(name) {
+  return `Hi, ${name}!`
 
```

Comment:
also, good! But where does the function end? (: You need a closing parenthesis to end the function body

```suggestion
function greet(name) {
  return `Hi, ${name}!`
}
```